src/Features/Demo/DemoGhostPlayer.cpp

```diff
@@ -284,19 +284,19 @@ CON_COMMAND(ghost_delete_by_ID, "ghost_delete_by_ID <ID>. Delete the ghost selec
     console->Print("Ghost %d has been deleted !\n", std::atoi(args[1]));
 }
 
-CON_COMMAND(ghost_delete_all, "Delete all ghosts.\n")
+CON_COMMAND(ghost_delete_all, "ghost_delete_all - delete all ghosts\n")
 {
     demoGhostPlayer.DeleteAllGhostModels();
     demoGhostPlayer.DeleteAllGhosts();
     console->Print("All ghosts have been deleted !\n");
 }
 
-CON_COMMAND(ghost_recap, "Recap all ghosts setup.\n")
+CON_COMMAND(ghost_recap, "ghost_recap - recap all ghosts setup\n")
 {
     demoGhostPlayer.PrintRecap();
 }
 
-CON_COMMAND(ghost_start, "Start ghosts")
+CON_COMMAND(ghost_start, "ghost_start - start ghosts")
```

Comment:
It would be pretty weird for commands with multiple lines of help, and I'm not sure whether it's better to leave it explicit or not, however I can probably just regex it this time, so lmk what you prefer

Reply:
I'll leave it for now - I'm just gonna fix this specific case myself real quick, then I'll merge this in